bin/p2-exec/main.go

```diff
@@ -114,6 +114,12 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
+
+	// setpgid: zeroes for current process ID
+	if err := unix.Setpgid(0, 0); err != nil {
```

Comment:
Behavior ought to be the same on FreeBSD. From the [BSD manpage](https://www.freebsd.org/cgi/man.cgi?sektion=2&query=setpgrp) I see:

```
If pgrp is zero,	then the process id of the process
specified by pid is used instead.
```